framework-v1/gov-framework.md

```diff
@@ -79,7 +98,7 @@ CoA can unban a member (remove `GovBan`flag) at any time to run for future sessi
 
 **Tenure per Term:** 6 months.
 
-**DAO Parameters:** `approval_threshold = 4, voting_duration = 5` , `cooldown = 7`
+**DAO Parameters:** `approval_threshold = 4, vote_duration = 5 days, min_vote_duration = 3 days, cooldown = 0 days`
```

Comment:
CoA doesn't have cooldown. In the contract config it is `0`. However here it has a wrong value (probably copy-paste error from HoM). CoA doesn't have power to create new proposals, it has power to veto.

Reply:
I do not see anything in these changes that changes the gov framework rather just clarify and fix typos. Looks good to me.